composer.json

```diff
@@ -75,9 +75,10 @@
         "coverage": "vendor/bin/phpunit --coverage-html build/reports/html --coverage-clover build/reports/clover.xml",
         "csfixer": "vendor/bin/php-cs-fixer fix --allow-risky=yes",
         "phpstan": "vendor/bin/phpstan analyse src tests --memory-limit=1G",
-        "rector": "vendor/bin/rector process --no-ansi",
+        "rector": "vendor/bin/rector process --no-ansi --no-diffs",
```

Comment:
don't show the diffs, as that can be quite verbose. if you want to see them you can run rector normally

Reply:
Do we really want this? The idea behind `composer qa` is local use. Aren't we likely to want diffs?

Reply:
I personally never run these kind of things through composer, but I can understand not everybody does this. I'll update it and update `composer qa` to use the command directly.

Reply:
Yeah I always use composer qa 😁